mltsp/predict.py

```diff
@@ -12,13 +12,29 @@
 from . import util
 
 
-def model_predictions(featureset, model):
-    """Construct a DataFrame of model predictions for given featureset."""
+def model_predictions(featureset, model, return_probs=True):
+    """Construct a DataFrame of model predictions for given featureset.
+
+    Parameters
+    ----------
+    featureset : xray.Dataset
+        Dataset containing feature values for which predictions are desired
+    model : scikit-learn model
+        Fitted scikit-learn model to be used to generate predictions
+    return_probs : bool, optional
+        Parameter to control the type of prediction made in the classification
+        setting (the parameter has no effect for regression models). If True,
+        probabilities for each class are returned where possible; if False,
+        only the predicted labels themselves are returned.
```

Comment:
Only the top class. Maybe it should be named something along those lines instead? Basically I just noticed that in my test case I kept repeating the same code to get the top prediction out of the list and thought it might be nice to have an option to just return that directly.